src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/caseupdated/CaseUpdatedAboutToSubmitHandler.java

```diff
@@ -160,6 +162,24 @@ && isNotBlank(appellant.getAddress().getPostcode())) {
         return preSubmitCallbackResponse;
     }
 
+    private void updateLanguage(SscsCaseData sscsCaseData) {
+        HearingOptions hearingOptions = sscsCaseData.getAppeal().getHearingOptions();
+        if (nonNull(hearingOptions)) {
+            DynamicList languageList = hearingOptions.getLanguagesList();
+
+            String language = null;
```

Comment:
Is having this be null okay or will it throw an error downstream / conflict with the display in ccd-defs / exui?

Reply:
No issue here, will make sure it won't get shown in the appeal details tab if the language has been removed